src/SvgXml.web.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { ViewProps } from 'react-native';
+import { View, unstable_createElement } from 'react-native-web';
+import { XmlProps } from './xml';
+type SvgXmlProps = XmlProps &
+  ViewProps & {
+    height: number | string;
+    width: number | string;
+    viewBox?: string;
+    preserveAspectRatio?: string;
```

Comment:
You need to add all props accepted by svg tag (fill, strokeWidth, etc)

Reply:
I added all props supported by the rn SvgXml component.